ckcp/hack/util/reset_ckcp.sh

```diff
@@ -0,0 +1,288 @@
+    if [[ -n "$pac_ns" ]];
+    then
+      kubectl delete ns "$pac_ns"
+    fi
+
+    printf "\n  Uninstalling Openshift-Pipelines Operator:\n"
+    # We start with deleting tektonconfig so that the 'tekton.dev' CRs are removed gracefully by it.
+    kubectl delete tektonconfig config
+    kubectl delete -k "$GITOPS_DIR/argocd/tektoncd"
+    openshift_pipelines_csv=$(kubectl get csv -n openshift-operators | grep -ie "openshift-pipelines-operator" | cut -d " " -f 1)
+    if [[ -n "$openshift_pipelines_csv" ]];
+    then
+      kubectl delete csv -n openshift-operators "$openshift_pipelines_csv"
+    fi
+    mapfile -t tekton_crds < <(kubectl get crd | grep -ie "tekton.dev" | cut -d " " -f 1)
+    if [[ "${#tekton_crds[@]}" -gt 0 ]];
+    then
+      for crd in "${tekton_crds[@]}"
+      do
+        kubectl delete crd "$crd" &
+        kubectl patch crd "$crd" --type json --patch='[ { "op": "remove", "path": "/metadata/finalizers" } ]' >/dev/null 2>&1
+      done
+    fi
+    openshift_pipelines_operator=$(kubectl get operator | grep -ie "openshift-pipelines-operator" | cut -d " " -f 1)
+    if [[ -n "$openshift_pipelines_operator" ]];
+    then
+      kubectl delete operator "$openshift_pipelines_operator"
+    fi
+
+    printf "\n  Uninstalling cert-manager Operator:\n"
+    kubectl delete -k "$PLNSVC_DIR/operators/cert-manager"
+    mapfile -t cert_manager_crds < <(kubectl get crd | grep -iE "cert-manager.io|certmanagers" | cut -d " " -f 1)
+    if [[ "${#cert_manager_crds[@]}" -gt 0 ]];
+    then
+      for crd in "${cert_manager_crds[@]}"
+      do
+        kubectl delete crd "$crd" &
+        kubectl patch crd "$crd" --type json --patch='[ { "op": "remove", "path": "/metadata/finalizers" } ]' >/dev/null 2>&1
+      done
+    fi
+    cert_manager_operator=$(kubectl get operator | grep -ie "cert-manager-operator" | cut -d " " -f 1)
+    if [[ -n "$cert_manager_operator" ]];
+    then
+      kubectl delete operator "$cert_manager_operator"
+    fi
+
+    # Checks if the operators are uninstalled successfully
+    mapfile -t operators < <(kubectl get operators | grep -iE "cert-manager-operator|openshift-gitops-operator|openshift-pipelines-operator" | cut -d " " -f 1)
+    if (( ${#operators[@]} >= 1 )); then
+        printf "\n\n[ERROR] Couldn't uninstall all Operators, please try removing them manually."
+        exit
+    fi
+    printf "\nAll the operators are successfully uninstalled.\n"
+}
+
+# Cleans ckcp deployed resources from the cluster
+uninstall_ckcp(){
+    # Extract label from synctargets inside the kcp user workspace to be able to select namespaces
+    # created in the compute cluster by the same kcp-workspace.
+    ns_label=$(KUBECONFIG=${KCP_KUBECONFIG} kubectl get synctargets.workload.kcp.dev -o yaml | yq '.items[0].metadata.labels' | grep -ie "internal.workload.kcp.dev/key")
+    if [[ -z $ns_label ]]; then
+        printf "\n\n[ERROR] Couldn't extract labels from kcp synctarget object.\n"
```

Comment:
Redirect the `ERROR` message to stderr with `>&2`, and use `exit 1`.
The helpful message can stay on stdout.